Data-Science/MLflow/bike-sharing-prediction.ipynb

```diff
@@ -11,6 +11,16 @@
     "### Deploy ML model using KServe inference service"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the profile namespace below\n",
+    "PROFILE_NAMESPACE = \"<profile-namespace>\""
```

Comment:
we should have to get profile_name through kubectl get ns and we should have to automate this.

Reply:
@ask664  I have updated the code to read the namespace from the configmap 'user-info-cm'.